src/main/org/deidentifier/arx/distributed/ARXDistributedAnonymizer.java

```diff
@@ -226,15 +224,26 @@ public ARXDistributedResult anonymize(Data data,
 
         if (!config.isPrivacyModelSpecified(EDDifferentialPrivacy.class) &&
              config.getMonotonicityOfPrivacy() != Monotonicity.FULL) {
+
+            int[] mergedSubsetIndicies = new int[subset.length];
+            int currentIndex = 0;
+            int addToValue = 0;
+            for (int i = 0; i<handles.size(); i++) {
+                for (int value : partitions.get(i).getSubset()) {
+                    mergedSubsetIndicies[currentIndex++] = value + addToValue;
+                }
+                addToValue = addToValue + partitions.get(i).getData().getNumRows();
+            }
```

Comment:
I just realized that maybe its not, due to partitioning, which changes the order of records. If so, please comment this here, because it is important to understand why the code is as it is.